CONTRIBUTING.md

```diff
@@ -110,4 +112,4 @@ If you are contributing to OpenRewrite on Windows, please ensure that you set `c
 git clone -c core.autocrlf=false https://github.com/openrewrite/rewrite.git
 ```
 
-Also, as mentioned in the [IntelliJ section](#intellij-changes) above, please ensure that your system uses `UTF-8` for character encoding.
\ No newline at end of file
+Also, as mentioned in the [IntelliJ section](#intellij-changes) above, please ensure that your system uses `UTF-8` for character encoding.
```

Comment:
I don't see any changes here but GitHub is telling me this line changed. 🤔

Reply:
I used the online editor, maybe that's why.. It has that effect sometimes. 🤷🏻